### PR TITLE
support multiple Insight servers usage

### PR DESCRIPTION
**Pull request by AlexanderPavlenko**

### CODE REVIEW

File: lib/straight-server/gateway.rb [hunk 1]
```diff
@@ -21,32 +21,47 @@ def initialize_exchange_rate_adapters
...
-
-        @blockchain_adapters <<
-          case name
-          when 'ChainCom'
-            adapter.mainnet_adapter(api_key_id: attrs['api_key_id'])
-          when 'Insight'
-            adapter.mainnet_adapter(main_url: attrs['mainnet_url'], test_url: attrs['testnet_url'])
-          else
-            adapter.mainnet_adapter
+      all_instances = {main: [], test: []}
```
Comment (savroff) [inline lib/straight-server/gateway.rb]: Some small things from style guide
spaces between brackets  `{ main: [], test: [] }`

Comment (AlexanderPavlenko) [inline lib/straight-server/gateway.rb]: That's not the rule I'm following. Style guides also suggest hashrocket syntax.

Comment (savroff) [inline lib/straight-server/gateway.rb]: ????
https://github.com/bbatsov/ruby-style-guide#hash-literals

```ruby
# bad
hash = { :one => 1, :two => 2, :three => 3 }

# good
hash = { one: 1, two: 2, three: 3 }
```

Comment (savroff) [inline lib/straight-server/gateway.rb]: it's pretty strange to follow own rules, when you work on open source for community)))

Comment (AlexanderPavlenko) [inline lib/straight-server/gateway.rb]: > Style guides also suggest hashrocket syntax

That was irony about things like https://github.com/styleguide/ruby. As you can see, they are subject to change.

> you work on open source for community

Wrong. I'm working on the product for end-users and I care about delivering features without bugs, not about opinions on where I should put spaces.

Comment (savroff) [inline lib/straight-server/gateway.rb]: Yes, they can change and ruby can change too) This is normal. :dancers:

**Pull request merged**